modules/meeting/lib/open_project/meeting/engine.rb

```diff
@@ -63,6 +63,17 @@ class Engine < ::Rails::Engine
            before: :members,
            icon: 'meetings'
 
+      #     if OpenProject::FeatureDecisions.more_global_index_pages_active?
+      #       menu :top_menu,
+      #            :meetings, { controller: '/meetings', project_id: nil, action: 'index' },
+      #            context: :modules,
+      #            caption: I18n.t('label_meetings_plural'),
+      #            if: Proc.new {
+      #              (User.current.logged? || !Setting.login_required?) &&
+      #                User.current.allowed_to_globally?(:view_meetings)
+      #            }
+      #     end
+
```

Comment:
I guess one option would be to add the check for the feature flag into the `if:` proc. That would be fine as well. But if anybody has another suggestion, please let me know.

Reply:
We could look into statically loading the feature decisions, but as it depends on `Settings`, this is not part of the initialization autoloading as that changed with Rails 7. I don't have a quick fix for that, so would vote for moving it into the proc.